src/desktop/apps/auction/reducers/__tests__/artworkBrowser.test.js

```diff
@@ -153,6 +153,22 @@ describe("auction/actions/artworkBrowser.test.js", () => {
         ])
         notAllArtworksFetched.artworkBrowser.total.should.eql(10)
       })
+
+      it("updates allFetched to true if the current page exceeds 100", () => {
+        const maxPageResponse = auctions(
+          {
+            artworkBrowser: {
+              total: 1,
+              saleArtworks: [],
+              filterParams: { page: 100 },
+            },
+          },
+          {}
+        )
+        const tooHighPage = auctions(maxPageResponse, actions.updatePage(false))
+        const notAllFetched = auctions(tooHighPage, actions.updateAllFetched())
```

Comment:
Do we need to explicitly pass on `false` like we do above? Is our goal here to stub redux state? Just trying to make sure I understand the test

Reply:
Yea, the `false` param for `updatePage` is what tells that API to increment the page, vs setting it to 1. It's a 'reset' argument: https://github.com/artsy/force/blob/cc12124d1b1982eac2a9d6b571d798a830ae55ac/src/desktop/apps/auction/reducers/artworkBrowser.js#L255

Reply:
gotcha, and the `const notAllFetched = auctions(tooHighPage, actions.updateAllFetched())` is the code actually being exercised in the test

Reply:
So just to be super clear, the `const tooHighPage = auctions(maxPageResponse, actions.updatePage(false))` gets us to the `101`st page, hence triggering the  `currentPage > 100` that you added, right?